ios/IJKMediaPlayer/IJKMediaPlayer/ijkmedia/ijksdl/ios/IJKSDLAudioQueueController.m

```diff
@@ -29,7 +29,7 @@
 
 #import <AVFoundation/AVFoundation.h>
 
-#define kIJKAudioQueueNumberBuffers (3)
+#define kIJKAudioQueueNumberBuffers (2)
```

Comment:
3 buffers is assumed in ffplay, if you want to change it from 3 to 2 you have to do a lot computational time logic.
And modify it has no benefit.